packages/image_picker/image_picker_for_web/lib/image_picker_for_web.dart

```diff
@@ -241,20 +250,20 @@ class ImagePickerPlugin extends ImagePickerPlatform {
   Future<List<XFile>> _getSelectedXFiles(html.FileUploadInputElement input) {
     final Completer<List<XFile>> _completer = Completer<List<XFile>>();
     // Observe the input until we can return something
-    input.onChange.first.then((event) {
+    input.onChange.first.then((event) async {
```

Comment:
There's no `await` inside this event function, so does this need to be `async`?

Reply:
done